src/View/Helper/PaginatorHelper.php

```diff
@@ -820,9 +822,10 @@ protected function _firstNumber($ellipsis, $params, $start, $options)
     protected function _lastNumber($ellipsis, $params, $end, $options)
     {
         $out = '';
+        $last = is_int($options['last']) ? $options['last'] : 0;
```

Comment:
So you are saying that inputting `1` vs `'1'` will lead to two different kind of behavior?
With PHP and not being typesafe this sounds quite a dangerous thing to do then. Or maybe I don't quite understand the issue here.